library/core/src/main/java/com/google/android/exoplayer2/audio/MediaCodecAudioRenderer.java

```diff
@@ -286,6 +287,19 @@ public MediaClock getMediaClock() {
     return this;
   }
 
+  @TargetApi(23)
+  @Override
+  protected void updateCodecOperatingRate(Format format) {
```

Comment:
I'm not sure there's any point doing anything for the audio case. Do you observe a difference? I strongly suspect there is none. Can we do video only for now, unless you can measure a change?